File: CSharp.Functions/Common/Entities/AffiliateDetails.cs
namespace Common.Entities
{
    public class AffiliateDetails
    {
        public string? Name { get; set; }
        public string? CompanyNumber { get; set; }
        public string? FirmReferenceNumber { get; set; }
        public bool isPRAAuthorised { get; set; } = false;
        public string? RegisteredAddress { get; set; }
        public string? EmailAddress { get; set; }
        public ContactNumber? ContactNumber { get; set; }
        public string? Website { get; set; }
        public string? TradingAddress { get; set; }
        public bool IsTradingAddressChanged { get; set; } = false;
    }
}

File: vue.jsonforms/src/entities/Affiliate/AffiliateDetails.ts
import { ContactNumber } from "@/entities/ContactNumber";

export class AffiliateDetails {
  public name: string | undefined;
  public companyNumber: string | undefined;
  public firmReferenceNumber: string | undefined;
  public isPRAAuthorised: boolean = false;
  public registeredAddress: string | undefined;
  public emailAddress: string | undefined;
  public contactNumber: ContactNumber | undefined;
  public website: string | undefined;
  public tradingAddress: string | undefined;
  public isTradingAddressChanged: boolean = false;
  public contactNumberDisplay?: string = "";
}